scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ -z "${SOURCECRED_GITHUB_TOKEN}" ]; then
```

Comment:
Does the command spit out a nice human error message if it's not provided? If so, great, and I'll remove this here.

Reply:
I'm going to leave this in for now - it can be a silent error if the user doesn't provide it (just tested)

```bash
$ docker logs sourcecred
fatal: SOURCECRED_GITHUB_TOKEN not set
fatal: run 'sourcecred help load' for help
yarn run v1.17.3
$ NODE_ENV=development webpack-dev-server --config config/webpack.config.web.js --host 0.0.0.0
(node:35) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
ℹ ｢wds｣: Project is running at http://0.0.0.0:8080/webpack-dev-server/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /code
```
It should not continue given the variable is missing.

Reply:
When the command is customized (e.g., scores added) we can do a check for that and act appropriately. For now it just serves to load and then run, so I'm leaving it.

Reply:
> I'm going to leave this in for now - it can be a silent error if the user doesn't provide it (just tested)
> 
> It should not continue given the variable is missing.

I think this is because you're not using the exit value of the `load` command. Try adding appending `|| exit 1` or using `&&` between load and start.

Reply:
okay trying this now.

Reply:
That does it! Since there are other commands that might be run, I'm going to remove load as well, and have the user provide it.

Reply:
I'll add instructions to the README about mounting the volume too.

Reply:
okey doke! Just added a complete example to mount the data (so it remains on the host) along with using a build argument, and I generalized the command so that the user doesn't have to load (but can do something else... something that I don't know how to do yet :P)